_data/accessibility-tests/header.yml

```diff
@@ -0,0 +1,218 @@
+    summary_additional: |
+      When navigating through the website,
+      you understand there are other options for navigation, such as Search or an HTML site map.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.5
+  - summary: You can hover over header content.
+    summary_additional: |
+      When you move the pointer over an active header menu item,
+      the header menu stays open.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.13
+  - summary: Header menu can be dismissed without moving the mouse or keyboard focus away.
+    summary_additional: |
+      When you press the escape key while the header menu is active,
+      the menu closes.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.13
+  - summary: Header menus stay open until they are dismissed.
+    summary_additional: |
+      Header menus close only when you move focus away or press the escape key.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.13
+# Zoom/screen magnification tests
+  - summary: Header links are visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom in to 200%,
+      header links are legible and properly sized.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.4
+  - summary: Header remains visible and functional when screen is magnified.
+    summary_additional: |
+      When you zoom in to 400%,
+      you can still see and use the header menu and navigation,
+      and it doesn't overlap with or get cut off by any other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Keyboard can access all elements in the header.
+    summary_additional: |
+      When you navigate through the header with a keyboard,
+      you can interact with every element inside it.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+  - summary: Header elements do not trap keyboard focus on desktop.
+    summary_additional: |
+      When you navigate with a keyboard in the desktop view,
+      you can easily move into and out of the header.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.2
+  - summary: Mobile menu navigation traps header focus.
+    summary_additional: |
+      When you access the header with a mobile device,
+      you will need to take action to close the menu before proceeding to another action.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.2
+  - summary: Keyboard can bypass header and navigation.
+    summary_additional: |
+      When you visit a page using the keyboard,
```

Comment:
This reads awkward to me on review. Suggesting but not attached to this suggestion:
"When you visit a page using the keyboard, you will find a “skip to main content” link at the top of the page, before the header navigation. The link allows you to navigate past the header and navigation into page content."

Reply:
Resolved in [ef25240](https://github.com/uswds/uswds-site/pull/2761/commits/ef25240656b6664eafa05f5a4fe9a1982c7f30f7)